src/EPR.PRN.Backend.Data/DataModels/PrnUpdateStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace EPR.PRN.Backend.Data.DataModels
+{
+    public class PrnUpdateStatus
+    {
+        public required string EvidenceNo { get; set; }
+        public required string EvidenceStatusCode { get; set; }
+        public DateTime? StatusDate { get; set; }
+        public required string AccreditationYear { get; set; }
```

Comment:
Unclear why AccreditationYear is required when its not in the json for update or bulk update PRN

Reply:
This is required as per the requirements to determine the status EV-AWACCEP             
or EV-AWACCEP-EPR depending on AccreditationYear